030-Concepts/075-file-attachments.mdx

```diff
@@ -120,6 +120,11 @@ img = xata.files().transform("https://us-east-1.storage.xata.sh/4u1fh2o6p10blbut
 
 ![Entire columns can be made public by default](images/column-access-public.png)
 
+<Alert status="warning">
+  ⚠️ *Public object caching*  
```

Comment:
Use the title prop on the Alert component for this. Remove the emoji.

Reply:
Already removed.
What is the title prop?

Reply:
No worries. this is fine.